src/prefect/server/api/flows.py

```diff
@@ -160,3 +161,59 @@ async def delete_flow(
         raise HTTPException(
             status_code=status.HTTP_404_NOT_FOUND, detail="Flow not found"
         )
+
+
+class FlowPaginationResponse(BaseModel):
```

Comment:
i think we might want to put this in `prefect.server.schemas.responses` for consistency